src/main/scala/Database/UserDatabase.scala

```diff
@@ -1,78 +1,150 @@
 package Database
 
 import Dao.DAO
+import Database.Connection.establishConnectionWithMySql
 import Model.User
+import Model.UserType.{Admin, Customer}
+import java.sql.ResultSet
 import java.util.UUID
-import scala.util.{Success, Failure}
-import scala.collection.mutable.ListBuffer
+import scala.annotation.tailrec
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
+import scala.util.{Failure, Success, Try}
+import org.slf4j.{Logger, LoggerFactory}
 
 class UserDatabase extends DAO {
 
-  private val userList = ListBuffer.empty[User]
+  private val connection = establishConnectionWithMySql()
+  private val logger: Logger = LoggerFactory.getLogger(getClass)
 
   def add(user: User): Future[String] = {
     Future {
-      Thread.sleep(100)
-      userList.addOne(user)
-      s"Added user ${user.name} to DB."
+      Try {
```

Comment:
This `TRY` can also be moved to a method as it is being used in all places in this file.